kong/plugins/http-log/handler.lua

```diff
@@ -68,9 +68,10 @@ local function parse_url(host_url)
   if not parsed_url.port then
     if parsed_url.scheme == "http" then
       parsed_url.port = 80
-
     elseif parsed_url.scheme == "https" then
       parsed_url.port = 443
+    else
```

Comment:
I think it would be better to give an error log here, because different schemas have different default ports. And the schema is missing here. Maybe it would be more reasonable for us to give a default schema?